huxley/core/models.py

```diff
@@ -180,6 +176,7 @@ def save(self, *args, **kwargs):
         if getattr(self, '_pending_country_preference_ids', []):
             self.update_country_preferences(self._pending_country_preference_ids)
             self._pending_country_preference_ids = []
+            post_save.connect(School.update_fees, sender=School)
```

Comment:
Why do we need to add this here? We've already hooked up `update_fees` to the `pre_save` signal below... why do we need to calculate it twice?